cluster/ci/config/prow/config.yaml

```diff
@@ -319,6 +319,15 @@ presubmits:
     rerun_command: "/test containerized"
     trigger: "((?m)^/test( all| containerized),?(\\s+|$))"
 
+  - name: test_pull_request_openshift_ansible_extended_conformance_install_crio
+    agent: jenkins
+    labels:
+      master: ci.openshift.redhat.com
+    always_run: true
+    context: ci/openshift-jenkins/containerized
```

Comment:
I think you want a different context here since this is already used by `test_pull_request_openshift_ansible_extended_conformance_install_containerized`.

Reply:
Yup, updated, thanks.